### PR TITLE
add(epp): barracuda bma snmp proc

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Description

-

## Target serie

- [X] 20.04.x
- [X] 20.10.x
- [X] 21.04.x
- [X] 21.10.x
- [X] 22.04.x (master)

### CODE REVIEW

**Review by lchrdn — APPROVED** [review]

File: en/integrations/plugin-packs/procedures/network-barracuda-bma-snmp.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+```bash
+/usr/lib/centreon/plugins/centreon_barracuda_bma_snmp.pl \
+    --plugin=network::barracuda::bma::snmp::plugin \
+    --mode=load \
+    --hostname='10.0.0.1' \
+    --snmp-version='2c' \
+    --snmp-community='my-snmp-community' \
+    --warning-cpu-load='' \
+    --critical-cpu-load='' \
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-barracuda-bma-snmp.md:126]: A value could help.

File: en/integrations/plugin-packs/procedures/network-barracuda-bma-snmp.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    --verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: system cpu load: 2.00% | 'system.cpu.load.percentage'=2.00%;;;0;100
+```
+
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/network-barracuda-bma-snmp.md:135]: A quick explanation of the threshold value.

File: fr/integrations/plugin-packs/procedures/network-barracuda-bma-snmp.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+l'utilisateur **centreon-engine**:
+
+```bash
+/usr/lib/centreon/plugins/centreon_barracuda_bma_snmp.pl \
+    --plugin=network::barracuda::bma::snmp::plugin \
+    --mode=load \
+    --hostname='10.0.0.1' \
+    --snmp-version='2c' \
+    --snmp-community='my-snmp-community' \
+    --warning-cpu-load='' \
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/network-barracuda-bma-snmp.md:125]: See EN review about command line example possible enhancments

**Review by Sims24 — COMMENTED** [review]
inline

**Review by Sims24 — CHANGES_REQUESTED** [review]
See comments inline. Thanks!

**Review by Sims24 — APPROVED** [review]

**Pull request merged**